provision/ubuntu/kernel-next.sh

```diff
@@ -32,9 +32,13 @@ make oldconfig && make prepare
 ./scripts/config --module CONFIG_TLS
 ./scripts/config --enable CONFIG_VBOXSF_FS
 
-sudo make -j$(nproc) deb-pkg
+make -j$(nproc) deb-pkg
 cd ..
 sudo dpkg -i linux-*.deb
 sudo ln -sf /boot/System.map-$(uname -r) /boot/System.map
 
+cd tools/perf
```

Comment:
This needs to be `cd k/tools/perf` or `cd $HOME/k/tools/perf`.

I'm wondering if these changes should be in `kernel-next-bpftool.sh` though (with a rename to e.g., `kernel-next-tools.sh`)? It seems closer in purpose.